src/Dfe.ContentSupport.Web/Views/Shared/RichText/_Vimeo.cshtml

```diff
@@ -1,7 +1,7 @@
 @model Hyperlink
 
 @{
-    var vimeoId = Model.Uri.Split('/').Last();
+    var vimeoId = Model.Uri.Split('/')[0];
```

Comment:
Same here

Reply:
Nope
[1aad58a](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/811/commits/1aad58a750b7b64bb5a326a9a35751aafb3ffbc6)